oag/ja-jp/content/topics/access-gateway/advanced-oag-policy-development.htm

```diff
@@ -125,7 +125,7 @@
                                 </div>
                             </div>
                             <div class="title-bar-layout outer-row">
-                                <div class="logo-wrapper"><a class="logo" href="https://support.okta.com/help/s/?language=ja" alt="Okta"></a>
+                                <div class="logo-wrapper"><a class="logo" href="https://help.okta.com/okta_help.htm?id=csh-index" alt="Okta"></a>
```

Comment:
Question about this redirect, would it be in Japanese? this is the same question I have on the other PRs

Reply:
Good catch! We need to update post process script for this link so that it will redirect to ja-jp.
I'll need to recreate PRs